### PR TITLE
add link to branching guide

### PR DESCRIPTION
**Pull request by starsplatter**

### CODE REVIEW

**Comment by maccabeelevine** [comment]
I like the idea to add that link, but the page it links to explains not just that bullet point but also duplicates several other bullet points.  Is it possible to combine the two documents rather than linking from one to the other?

**Review by craigmcnally — APPROVED** [review]

**Comment by craigmcnally** [comment]
> I like the idea to add that link, but the page it links to explains not just that bullet point but also duplicates several other bullet points. Is it possible to combine the two documents rather than linking from one to the other?

We can always make adjustments on the wiki side if needed.

**Comment by starsplatter** [comment]
I updated this a little to try and clarify what I was hoping to do - tell them how to set up their first PR and then refer to the wiki for everything else.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ For process overview, please see https://wiki.folio.org/display/TC/RFC+Process
 * Fork the official RFC repo at https://github.com/folio-org/rfcs . This is usually only needed once for any number of RFCs to be submitted.
 * Create a file in the text directory at the top of the forked repo. It should follow the template defined here. Ensure that the file is named appropriately, taking note of the sequence numbers of existing RFCs.
```
Comment (maccabeelevine) [inline README.md:8]: These two bullets are (I think) word-for-word duplicated on the wiki page.

**Review by maccabeelevine — COMMENTED** [review]
I like all of @starsplatter 's edits and would be fine to approve this.  I don't really understand though how the link is meant to

> refer to the wiki for everything else

since there is still direct duplication between this and the wiki page.

I am fine with @craigmcnally 's comment that 

> We can always make adjustments on the wiki side if needed.

if that's also @starsplatter 's perspective.

File: README.md (hunk 1 above)
Comment (starsplatter) [reply README.md:8]: This duplication is intentional in my mind. It is here to provide a complete set of directions for the first stage, while linking back to the wiki for further information needed later in the process.

Comment (starsplatter) [reply README.md:8]: quick start guide vs full documentation essentially?

**Pull request merged**